deploy/3000_deploy_asset_giveaway_1.ts

```diff
@@ -1,7 +1,7 @@
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
 import {DeployFunction} from 'hardhat-deploy/types';
-import getAssets from '../data/asset_giveaway_1/getAssets';
-import * as assetData from '../data/asset_giveaway_1/assets.json';
+import {createAssetClaimMerkleTree} from '../data/asset_giveaway_1/getAssets';
+import {default as assetData} from '../data/asset_giveaway_1/assets.json';
```

Comment:
is that we had to do to make it work.
`import assetData from '../data/asset_giveaway_1/assets.json';` did not work ?

Reply:
Yes exactly!